src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/BlockWithData.java

```diff
@@ -46,4 +46,7 @@ public BlockWithData(EthBlock.Block ethBlock) {
         this.timestamp = ethBlock.getTimestamp();
     }
 
+    public List getTransactions() {
```

Comment:
Missing [Generic Type](https://docs.oracle.com/javase/tutorial/java/generics/types.html).
Type is already missing on the field though.

Note: Classes from `org.web3j` should not be exposed as part of the API if possible (hinting at the question what will be the generic type).

Reply:
The branch from the PR is a WIP one, the main idea of the commit is adding a new method in Bloxberg client to send an object of BlockWithData instead of EthBlock (from org.web3j) to whoever asks for a Block. As regards the method getTransactions, it is a WIP method just to have a working version of the code. The final version will be an object of BlockTransaction class (a custom class defined by us). However, I can change it to be generic, and for now for practical purposes send the transactions from the Ethereum block.

Reply:
If current state of the PR is a WIP and review is not called for, then please change PR to draft (there is a button for this in Github, or when creating a PR).

Also, I am not sure if you understood correctly what I said when referring to Generic Types.
We are talking about changing the return type to `List<EthBlock.Transaction>` or something alike, yes? Raw types for generic typed classes should never be used in Java.